utils.go

```diff
@@ -9,33 +9,46 @@ import (
 
 var (
 	timestampType = reflect.TypeOf(Timestamp{})
-	Facilities    = []string{
+
+	// Facilities DEPRECATED See Facilities.List
```

Comment:
I looked at Facilities.List and saw nothing mentioned there.
We have the list of facilities here to speed up test for whether a facility code is correct.

Reply:
dtto for All the other resource listings.

Reply:
I'll change "See" to "Use".  I didn't mean for users to find more explanation in the Facilities.List documentation, rather I meant they should not use this static list in favor of the dynamic one.

If we need this for packngo tests we should put it in `utils_test.go` (or more likely `packngo_test.go`). FWIW, I didn't see `Facilities` being referenced at all in this project.

If non-packngo packages are using this variable it's more problematic, because this is an outdated and incomplete list.